monitoring/nagios.py

```diff
@@ -383,6 +396,48 @@ def _fmt_dt_str(self, cmd, host, duration, author=None,
 
         return dt_str
 
+    def _fmt_dt_del_str(self, cmd, host, svc=None, start=None, comment=None):
+        """
+        Format an external-command downtime string.
```

Comment:
*nit pick* that should say

> Format an external-command downtime deletion string.

As it is different from `_fmt_dt_str()` which uses the same docstring. Not a blocker.